docs/upgrade.html

```diff
@@ -151,7 +151,12 @@ <h5><a id="upgrade_110_notable" href="#upgrade_110_notable">Notable changes in 1
 </ul>
 
 <h5><a id="upgrade_110_new_protocols" href="#upgrade_110_new_protocols">New Protocol Versions</a></h5>
-<ul></ul>
+<ul>
+    <li> <a href="https://cwiki.apache.org/confluence/display/KAFKA/KIP-226+-+Dynamic+Broker+Configuration">KIP-226</a>: DescribeConfigsRequest v1 introduces a <code>include_synonyms</code> field. </li>
```

Comment:
I wonder how useful it is to mention these fields individually since you really have to read the KIP to understand the usage. Maybe we could just mention one point for the DescribeConfigsRequest and one point for the FetchRequest and point users to the respective KIPs?